src/odemis/gui/win/acquisition.py

```diff
@@ -103,15 +106,20 @@ def __init__(self, parent, orig_tab_data):
 
         # Compute the preset values for each preset
         self._preset_values = {}  # dict string -> dict (SettingEntries -> value)
-        orig_entries = get_global_settings_entries(self._settings_controller)
+        self._orig_entries = get_global_settings_entries(self._settings_controller)
         for sc in self.streambar_controller.stream_controllers:
-            orig_entries += get_local_settings_entries(sc)
-        self._orig_settings = preset_as_is(orig_entries) # to detect changes
+            self._orig_entries += get_local_settings_entries(sc)
+        self._orig_settings = preset_as_is(self._orig_entries)  # to detect changes
         for n, preset in presets.items():
-            self._preset_values[n] = preset(orig_entries)
+            self._preset_values[n] = preset(self._orig_entries)
         # Presets which have been confirmed on the hardware
         self._presets_confirmed = set() # (string)
 
+        # Get all the VA's from the stream and subscribe to them for changes.
```

Comment:
This means we don't need to use the pubsub events `setting.changed anymore, right? If so, let's drop it (l. 267 and 416).

Reply:
Removed - seems like it isn't causing any trouble yet.

Reply:
Let's remove every place related to this event then. See:
`grep -Ir "setting.changed" src/odemis/gui/`